internal/abci/accumulator.go

```diff
@@ -20,6 +20,7 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/encoding"
+	"github.com/tendermint/tendermint/libs/log"
```

Comment:
Why use tendermint's logging?  This has sometimes been an issue in the past for me, using basic functionality from a package that isn't providing that functionality as a core value.